crates/solidity/testing/snapshots/cst_output/ContractDefinition/member_constructor_definition/generated/0.4.22-success.yml

```diff
@@ -1,27 +0,0 @@
-# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
-
-Source: >
-  1  │ contract Sample {                                                                │ 0..17
-  2  │   constructor() public {                                                         │ 18..42
-  3  │   }                                                                              │ 43..46
-  4  │ }                                                                                │ 47..48
-
-Errors: []
-
-Tree:
-  - ContractDefinition (Rule): # 0..48 "contract Sample {\n  constructor() public {\n  }\n}"
-      - ContractKeyword (Token): "contract" # 0..8
```

Comment:
I think this should still succeed in `0.4.22`, right. Is this a regression?

Reply:
Yes, since #568 is not yet implemented. I've addressed this in 8f3e978cfb9f39f9c46546afc42c0cc7a081f9d4 to try and preserve the current behaviour.